### ISSUE TITLE
Error on build

### ISSUE THREAD

**Issue by samburgers** [birkir/gatsby-source-prismic-graphql#58]
First time looking at this source plugin, seem to be getting this error on build

```
 ERROR #11321  PLUGIN

"gatsby-source-prismic-graphql" threw an error while running the sourceNodes lifecycle:

The "options.agent" property must be one of type Agent-like Object, undefined, or false. Received type function
```

## Steps to reproduce:

- `cd /examples/static-query`
- `npm install`
- `npm run develop`

thanks!

**Comment by ghost** [comment]
Also having this exact issue. Have been looking for solution for few hours now. Anyone has any idea?

**Comment by samburgers** [comment]
Not sure if related, but there appears to be a similar error over here
https://github.com/axios/axios/issues/1627

**Comment by samburgers** [comment]
Ok, looks like we might need to force revert `prismic-javascript` down to `2.0.3` like here:

https://github.com/prismicio/prismic-javascript/issues/92

**Comment by ghost** [comment]
yeah reverting it works, thanks man

**Comment by arnaudlewis** [comment]
It's fixed in `prismic-javascript@2.1.1`. Sorry for the inconvenience.
